copilot/fastapi/run_tests.sh

```
@@ -0,0 +1,85 @@
+# Run the tests
+if [ -z "$SPECIFIC_TEST" ]; then
+    echo -e "${YELLOW}Running all tests...${NC}"
+    # Add -s to show print outputs which can help with debugging
+    python -m pytest tests/ $VERBOSE $COVERAGE -s
+else
+    echo -e "${YELLOW}Running specific test: $SPECIFIC_TEST${NC}"
+    python -m pytest $SPECIFIC_TEST $VERBOSE $COVERAGE -s
+fi
+
+# Check the test result
+if [ $? -eq 0 ]; then
```

Shellcheck (0.10.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)